scripts/install_python_toolchain.sh

```diff
@@ -3,7 +3,7 @@
 set -exo pipefail
 
 if [[ -z $VIRTUALENV ]]; then
-  VIRTUALENV=virtualenv
+  VIRTUALENV='python3 -m venv'
```

Comment:
We still need to support Python 2.7. I don't think this is going to work for Python 2.7